### ISSUE TITLE
 Dust Coins issue 

### ISSUE THREAD

**Issue by engAhmad** [TeamWertarbyte/crypto-trading-bot#40]
Thanks for the invite, joined!

Dust are all assets that are in your portfolio but only worth a few bucks. Like 0.24$ of Digibyte.
Currently the bot sees it and thinks "Well we still got some, so I don't need to buy more" or it tries to sell it due to bad EMA but can't due to minimum trading rate being greater of what you hold.
I had like 4-5 dust assets that "blocked" further transactions. But corrected them manually now.
I think with the current implementation we shouldn't get any more dust coins as we use fill_or_kill for selling. So there is no "partially sold" assets
Tldr: Fixed it manually and we shouldn't care about it anyway I think

_Originally posted by @saschb2b in https://github.com/TeamWertarbyte/crypto-trading-bot/issues/37#issuecomment-782868059_
